### DISCUSSION TITLE
Support multi-arch Docker builds

### DISCUSSION THREAD

**Discussion by kculmback-eig** [jef/streetmerchant#2234] [Ideas and feature requests]
Trying to run the Streetmerchant Docker image on an arm based computer (such as a Raspberry Pi) results in an error:

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm/v7) and no specific platform was requested
standard_init_linux.go:219: exec user process caused: exec format error
```

Specifying a platform does not help:

```
Error response from daemon: image with reference ghcr.io/jef/streetmerchant:nightly was found but does not match the specified platform: wanted linux/arm/v7, actual: linux/amd64.
```

In theory, it should be pretty easy to update the docker build commands to use docker buildx, which supports multi-arch builds: https://docs.docker.com/docker-for-mac/multi-arch/

**Comment by bmrussell** [comment]
I don't think Alpine, the base image support armv7 (?) and only just found out about buildx, so in the meantime I just submitted a PR for an dockerfile that works on rpi.
